pkg/operators/formatters/formatters.go

```diff
@@ -123,6 +129,30 @@ type replacer struct {
 
 // careful: order and priority matter both!
 var replacers = []replacer{
+	{
+		name:      "signal",
+		selectors: []string{"type:" + SignalTypeName},
+		replace: func(ds datasource.DataSource, in datasource.FieldAccessor) (func(data datasource.Data) error, error) {
+			out, err := ds.AddField(in.Name() + "_str")
```

Comment:
Are we sure about "_str" now? The other replacers don't use it anymore (they had easy exits, though, hehe.)

Also, maybe add the possibility to change the target name using an annotation like done with "formatters.timestamp.target". That served more as an example, but I think it adds value.

Reply:
I changed the approach a bit. Now the original column is rewritten to contain the signal name, after all the string representation is the most interesting one. The raw value is still kept on a `foo_raw` column. 

Later on we can optimize it to avoid the value to string conversion in case the consumer is not interested in it.